lua/vgit/ui/screens/CodeScreen.lua

```diff
@@ -273,7 +273,7 @@ function CodeScreen:make_line_numbers()
         and (lnum_change.type == 'remove' or lnum_change.type == 'void')
```

Comment:
Just need to run stylua on this file and we are all set to have this merged.

Reply:
Should be good now